pkg/core/native/policy.go

```diff
@@ -475,16 +476,21 @@ func (p *Policy) unblockAccount(ic *interop.Context, args []stackitem.Item) stac
 	return stackitem.NewBool(true)
 }
 
-func (p *Policy) getUint32WithKey(dao dao.DAO, key []byte) uint32 {
+func (p *Policy) getUint32WithKey(dao dao.DAO, key []byte) (uint32, error) {
```

Comment:
What if we're to have
```
func (p *Policy) getUint32WithKey(dao dao.DAO, key []byte, default uint32) uint32
```
I think it'd simplify a lot of code using this function.

Reply:
Fixed, looks prettier now.